WordPress/WordPressTest/WPAvatarSourceTest.m

```diff
@@ -48,7 +48,7 @@ - (void)testGravatarCaches
         XCTAssertNotNil(image, @"avatar should be downloaded");
         XCTAssertEqual(image.size.width, 48.f, @"avatar should be resized");
     }];
-    [self waitForExpectationsWithTimeout:5.0 handler:nil];
+    [self waitForExpectationsWithTimeout:30.0 handler:nil];
```

Comment:
Actually, this was totally unrelated. Up in that class we're mocking the response, so... i'll revert this commit, thanks!